app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/SearchRequestAutoMapperProfile.cs

```diff
@@ -80,4 +87,46 @@ public class IdentifierTypeConverter : IValueConverter<int?, int?>
         }
     }
 
+    public class PersonalIdentifierTypeConverter : IValueConverter<PersonalIdentifierType, int?>
+    {
+        public int? Convert(PersonalIdentifierType source, ResolutionContext context)
+        {
+            return source.ToDynamicIdentifierType();
+        }
+    }
+
+
+
+    public static class SearchApiIdentifierType
+    {
+        public static int? ToDynamicIdentifierType(this PersonalIdentifierType value)
+        {
+            switch (value)
+            {
+                case PersonalIdentifierType.DriverLicense:
+                    return IdentificationType.DriverLicense.DynamicValue;
+                case PersonalIdentifierType.SocialInsuranceNumber:
+                    return (int)IdentificationType.SocialInsuranceNumber.DynamicValue;
+                case PersonalIdentifierType.PersonalHealthNumber:
+                    return (int)IdentificationType.PersonalHealthNumber.DynamicValue;
+                case PersonalIdentifierType.BirthCertificate:
+                    return (int)IdentificationType.BirthCertificate.DynamicValue;
+                case PersonalIdentifierType.CorrectionsId:
+                    return (int)IdentificationType.CorrectionsId.DynamicValue;
```

Comment:
there is an issue with this object, `Dynamic` (should be `Dynamics`) value is duplicated in the object, you should use `.Value()` and it's already an integer, so no need to cast.
On the other end this mapping is cleaner than the mix in the IdentificationType class. we should clean it up in a following pr

Reply:
Identification Type represents the `dynamics context` it should not know about the searchApi